website/content/en/docs/helm/reference/advanced_features.md

```diff
@@ -77,6 +77,8 @@ To warn users that their CR settings may be ignored, the Helm operator creates e
 the CR that include the name and value of each overridden value. For example:
 
 ```
+$ kubectl describe Nginx -n nginx-operator-system 
+...
```

Comment:
This command is specific to the `Nginx` kind, which isn't mentioned anywhere else in the "Advanced Features" page. What does the output of `kubectl get events` look like?
```suggestion
$ kubectl get events
```

Reply:
The explanation is `To warn users that their CR settings may be ignored, the Helm operator creates events on
the CR that include the name and value of each overridden value.` so I think we should keep with the describe. IMO is nice know that is possible to check it in the CR.